src/controllers/controllerengine.h

```diff
@@ -107,6 +107,7 @@ class ControllerEngine : public QObject {
     Q_INVOKABLE void scratchDisable(int deck, bool ramp = true);
     Q_INVOKABLE bool isScratching(int deck);
     Q_INVOKABLE void softTakeover(QString group, QString name, bool set);
+    Q_INVOKABLE void softTakeoverResync(QString group, QString name);
```

Comment:
I mean ignored as in the control is warped to the new value.  like "oh resync, that's a way to override soft takeover".

reset makes it a little clearer I think